### ISSUE TITLE
Add "Show full remote usernames" option

### ISSUE THREAD

**Issue by dariusk** [hometown-fork/hometown#1247]
On Friend Camp we have a custom checkbox in user settings, "Show full username (including domain) for remote users".

![image](https://user-images.githubusercontent.com/266454/208766305-2e81e917-59ca-4ddc-beaa-c9f50e2882b6.png)

This is optional and lets users make it so remote usernames always contain the @server.name on the end, and local usernames do not. So if I am having a conversation between @dave@example.server and @dave@friend.camp (my own server) it will look like this:

```
@dave @dave@example.server hi Daves, I know which is which just by glancing here
```

Local conversations look like:

```
@dave @john @jade @rose hi everyone, it sure is nice on this server with you all
```

Mixed/remote conversations look like:

```
@jake@alpha.server @roxy@alpha.server hi there, I hope it is nice over on alpha server
```

Basically this gives users the option to make it explicit that they are interacting with remote accounts where that is true, and it provides a nice visual indicator for context of remote vs local vs mixed conversation.
